src/table/utils/handle-key-press.ts

```diff
@@ -111,6 +114,9 @@ export const handleHeadKeyDown = ({
     case 'Enter':
       changeSortOrder(layout, column);
       break;
+    case 'c':
+      isFlagEnabled && copyCellValue(cellValue);
```

Comment:
check for ctrl/meta here?

Reply:
Your tests are still passing, so wouldn't hurt to add a test to see that it is not called when ctrl/meta is not pressed.